Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Effectiveness.mo

```diff
@@ -0,0 +1,155 @@
+within Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses;
+model Effectiveness
+  "Model for calculating the heat exchange effectiveness"
+  extends Modelica.Blocks.Icons.Block;
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCooPL(final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHea_nominal(final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHea_nominal(final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHeaPL(final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHeaPL(final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.VolumeFlowRate VSup_flow_nominal(
+    final min = 100*Modelica.Constants.eps)
+    "Nominal supply air flow rate";
+
+  Modelica.Blocks.Interfaces.RealInput TSup(
+    final min=0,
+    final unit="K",
+    final displayUnit="degC")
+    "Supply air temperature"
+    annotation (Placement(transformation(extent={{-140,-60},{-100,-20}})));
+  Modelica.Blocks.Interfaces.RealInput TExh(
+    final min=0,
+    final unit="K",
+    final displayUnit="degC")
+    "Exhaust air temperature
+    " annotation (Placement(transformation(extent={{-140,-100},{-100,-60}})));
+  Modelica.Blocks.Interfaces.RealInput VSup_flow(final unit="m3/s")
+    "Supply air volumetric flow rate"
+    annotation (Placement(transformation(extent={{-140,60},{-100,100}})));
+  Modelica.Blocks.Interfaces.RealInput VExh_flow( final unit="m3/s")
+    "Exhaust air volumetric flow rate"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}})));
+  Modelica.Blocks.Interfaces.RealInput uSpe(final unit="1") "Wheel speed"
```

Comment:
Should be wheel speed have contributes (`min=0, max=1`)?

Reply:
I added <code>max=1</code>. However, since there may be minimum speed ratio requirement, I didn't add <code>min=1</code>.